MAUI/Circular-Charts/DataLabels.md

```diff
@@ -93,4 +93,133 @@ chart.Series.Add(series);
 
 {% endtabs %}
 
-![Applying series interior for data label in MAUI chart](DataLabel_images/maui_chart_datalabels_useseriespalette.png)
\ No newline at end of file
+![Applying series interior for data label in MAUI chart](DataLabel_images/maui_chart_datalabels_useseriespalette.png)
+
+## Formatting Label Context
+
+The content of the label can be customized using the [LabelContext]() property. Following are the two options that are supported now,
+
+* [Percentage]() - This will show the percentage value of corresponding data point Y value
+
+* [YValue]() - This will show the corresponding Y value.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+        
+        <chart:PieSeries ItemsSource="{Binding Data}" 
+                         LabelContext="Percentage"
+                         ShowDataLabels="True"
+                         XBindingPath="Product" 
+                         YBindingPath="SalesRate"/>  
+        
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+    . . .
+    PieSeries series = new PieSeries()
+    {
+        ItemsSource = new ViewModel().Data,
+        XBindingPath = "Product",
+        YBindingPath = "SalesRate",
+        ShowDataLabels = true,
+        LabelContext = LabelContext.Percentage
+    };
+
+    chart.Series.Add(series);
+    this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_datalabel_context.png)
+
+## LabelTemplate
+
+The [SfCircularChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html) provides support to customize the appearance of the datalabel by using the [LabelTemplate]() property.
```

Comment:
The [SfCircularChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html) provides support to customize the appearance of the datalabel using the [LabelTemplate]() property.

Reply:
Content updated.